docs/canarytokens/actions.md

```diff
@@ -257,4 +548,63 @@ These are a collection of endpoints that allow you mint new, interact with, and
 
 <APIDetails :endpoint="$page.frontmatter.endpoints.update"/>
 
+#### Example
+
+
+:::: tabs :options="{ useUrlFragment: false }"
+
+::: tab "cURL"
+
+``` bash
+curl https://EXAMPLE.canary.tools/api/v1/canarytoken/update \
+  -d auth_token=EXAMPLE_AUTH_TOKEN \
+  -d canarytoken=EXAMPLE_CANARY_TOKEN \
+  -d memo=EXAMPLE_MEMO 
```

Comment:
Same as above, `EXAMPLE_CANARYTOKEN` and for the memo we can make it `Updated example memo` to reinforce the update action.